config/puma.rb

```diff
@@ -21,7 +21,7 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+workers ENV.fetch("WEB_CONCURRENCY") { 3 }
```

Comment:
I'm 100% sure we want the Puma 5 update.

We will have to look at enabling multiple workers _very closely_ to avoid breaking the site, or causing strange intermittent bugs. We have a fairly low (manageable) volume of website traffic much of the time, also.

I'd be inclined to merge this pull request _without enabling multiple workers_, and put it up on a staging branch with multiple workers enabled for about a month or two.

Not many person-hours are behind maintaining this web app at the moment. The web app also underpins the Android and (deprecated) iOS apps; The API server is part of the web app. So the whole of "Refuge Restrooms" stops working if our site stops working. And properly validating that this causes _no bugs at all_ is a bit beyond my skills. Hence I am highly nervous with regard to enabling concurrency.

Reply:
Ok, I understand the situation, so I'll update this PR to just update the puma, and I'll submit another with just the concurrency part, then we could deploy step by step :)

Reply:
Thanks.

That would work very well.

I can merge the Puma update, and deploy that to the main site. Then activate multiple workers on the staging site for about a month or two. If all goes well, then the multiple workers change can be deployed to the main site as well.

Reply:
@DeeDeeG done!

I've updated this PR and created the https://github.com/RefugeRestrooms/refugerestrooms/pull/654 hope we can merge it now :)